src/VscPreviewExtension/package.json

```diff
@@ -2,23 +2,36 @@
   "name": "DfmPreview",
   "displayName": "DfmPreview",
   "description": "Preview the DocFX Flavored Markdown and token tree in Visual Studio Code",
-  "version": "0.0.21",
-  "publisher": "928pjy",
+  "version": "0.0.1",
```

Comment:
I am going to publish a version in v0.0.1, if everything is ok,it will be update to the same version with README. Is it ok?